ui/components/ui/list-item/list-item.component.js

```diff
@@ -33,16 +33,18 @@ export default function ListItem({
         }
       }}
     >
-      {icon ? <div className="list-item__icon">{icon}</div> : null}
-      <div className="list-item__heading">
-        {React.isValidElement(title) ? (
-          title
-        ) : (
-          <h2 className="list-item__title">{title}</h2>
-        )}
-        {titleIcon && (
-          <div className="list-item__heading-wrap">{titleIcon}</div>
-        )}
+      <div className="list-item__topWrap">
```

Comment:
```suggestion
      <div className="list-item__top-wrap">
```
- To be consistent with the coding style

Reply:
fixed in a78ec32a4291acbd25e7479c9891d86dfb1b8c82